relay-general/src/protocol/contexts/browser.rs

```diff
@@ -1,4 +1,7 @@
+use crate::protocol::FromUserAgentInfo;
+use crate::store::user_agent::{get_version, is_known};
 use crate::types::{Annotated, Object, Value};
+use crate::user_agent::{parse_user_agent, ClientHints};
```

Comment:
When looking on the usage these imported functions, it's sometimes not clear where they come from. It would be nicer if we always specify the module when the call the function , e.g. `user_agent::get_version()`